### PR TITLE
Use Element instead of riot.im in some links

### PR DESCRIPTION
**Pull request by SimonBrandner**
Screenshot_20210416_145325

### CODE REVIEW

File: gatsby/content/projects/clients/element.mdx [hunk 1]
```diff
@@ -58,7 +58,7 @@ features:
 
 [Element](https://element.io) is a glossy Matrix client for the web built on top of [matrix-react-sdk](https://matrix.org/docs/projects/sdk/matrix.org-react-sdk.html) with an emphasis on performance and usability.
 
-You can use it at [https://app.element.io](https://app.element.io), read more at [https://riot.im](https://riot.im) and get the source from [GitHub](https://github.com/vector-im/vector-web)!
+You can use it at [https://app.element.io](https://app.element.io), read more at [https://element.io/](https://element.io) and get the source from [GitHub](https://github.com/vector-im/element-web)!
```
Comment (MadLittleMods) [inline gatsby/content/projects/clients/element.mdx:61]: Minor nit: One difference I see is the trailing slash on `https://element.io/` 🤷

**Comment by MadLittleMods** [comment]
Overall looks good and we can merge :+1:

http://localhost:8000/docs/projects/client/element

**Comment by MadLittleMods** [comment]
Thanks for the updates @SimonBrandner 🚀

**Comment by SimonBrandner** [comment]
No problem!

**Pull request merged**